modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2258,6 +2259,65 @@ public void testDeleteObjectEntry() throws Exception {
 			publishedObjectDefinition);
 	}
 
+	@Test
+	public void testGetErrorLabel() throws Exception {
```

Comment:
Hey @rijema can we move to `ObjectValidationRuleLocalServiceTest`?

Reply:
Done!